temba/flows/migrations/0112_flow_ussd_push_enabled.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-06-16 13:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('flows', '0111_auto_20170824_1458'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='flow',
+            name='ussd_push_enabled',
+            field=models.BooleanField(default=False, help_text='Whether USSD Push is enabled for this flow'),
```

Comment:
I'm not so familiar with the USSD side of things so bear with me - but why is this a property of a flow rather than a channel?